crm_lead_to_task/wizard/crm_lead_convert2task.py

```diff
@@ -36,7 +36,6 @@ def action_lead_to_project_task(self):
         vals = {
             "name": lead.name,
             "description": lead.description,
-            "email_from": lead.email_from,
```

Comment:
Isn't this field now called `email_cc`?

Reply:
Yes, email_from dissappears. email_cc is used above, check https://github.com/OCA/crm/pull/584/commits/9a1f0aaf542b12882bbfc598fd055e656173ce74#diff-e7c057e506efe24fd0e6c307c1c712375b6d262f8919b0bb05851d275a93ba05R19